instructions.md

```diff
@@ -4,36 +4,37 @@
 
 1. Config:
 
-* Mediasources - Nextcloud is the default media storage Jellyfin will use as a dependency for media. If you use Filebrowser to store media, you will need to add this as a Media Source. 
+- Mediasources - Nextcloud is the default media storage Jellyfin will use as a dependency for media. If you use Filebrowser to store media, you will need to add this as a Media Source.
 
-* Plugins - Jellyfin supports plugins for Chromecast, which requires downloading files from Google servers to support this functionality, as well as YoutubePlayer, which autoloads youtube movie trailers matching media metadata. These plugins have been disabled by default. If you would like to use these features, toggle the corresponding switch to 'on'. Save any Config changes.
+- Plugins - Jellyfin supports plugins for Chromecast, which requires downloading files from Google servers to support this functionality, as well as YoutubePlayer, which autoloads youtube movie trailers matching media metadata. These plugins have been disabled by default. If you would like to use these features, toggle the corresponding switch to **on**. Save any Config changes.
 
-2. Launch the Jellyfin UI, select 'Next', and choose a username and password. We recommend saving this password in a password manager, such as Vaultwarden on your Start9 server. Press 'Next'.
+2. Launch the Jellyfin UI, select **Next**, and choose a username and password. We recommend saving this password in a password manager, such as Vaultwarden on your Start9 server. Press **Next**.
 
 3. Add a Media Library:
 
-* Select 'Add Media Library' followed by the 'Content Type' and 'Display Name' for the library being added. Click the `+` button to the right of 'Folders' and choose the corresponding directory where your media is stored. Jellyfin can support both filebrowser `/mnt/filebrowser` as well as nextcloud `/mnt/nextcloud`.
+- Select **Add Media Library** followed by the **Content Type** and **Display Name** for the library being added. Click the `+` button to the right of **Folders** and choose the corresponding directory where your media is stored. Jellyfin can support both filebrowser `/mnt/filebrowser` as well as nextcloud `/mnt/nextcloud`.
```

Comment:
This is the most confusing bit, would be awesome to get a community tech to make a 2min video of a jellyfin set up on Nextcloud